Remove.py

```diff
@@ -1,35 +1,48 @@
 import PrintList
 
 
-def remove_1(to_Do, removed):
+def remove_1(to_Do):
 
   print("""
   ------------------------------
   Remove an item
   ------------------------------
   """)
-  
+
+  # Print current list
   PrintList.print_list(to_Do)
 
-  X = input("[Enter X to exit this screen]")
+  #User presses X to exit screen
+  X = input("[Enter X to exit this screen]").upper()
 
-  item_to_remove = int(input("What item would you like to remove?")) - 1
+  if X == "X":
+    return
+  else:
+    pass
 
-  
-  # Calling the item by index and not by name
-  if item_to_remove <= len(to_Do):
+  for item_to_remove in range(3):
+    
+    item_to_remove = input("What item would you like to remove?").upper()
 
-    user_choice = to_Do[item_to_remove]
+    if item_to_remove == "X":
+      return
 
-    to_Do.remove(user_choice)
-    removed.append(user_choice)
+    try:
+      task = int(item_to_remove) - 1
+      if 0 <= task <= len(to_Do):
+        user_choice = to_Do[task]
+        to_Do.remove(user_choice)
+        PrintList.print_list(to_Do)
+        return
 
-  elif X == "X":
-    return
+      else:
+        print("Enter a number within range!")
+        
 
-  else:
+    except ValueError:
+      print("Please enter a valid number!")
 
-      print("Item not listed")
-      
-  PrintList.print_list(to_Do)
+    else:
+      print("Enter number within range!")
```

Comment:
We're printing the same message on line 39. Is there a way to simplify the code so we're only checking the validity of the number one time and printing the error message in only one place?